src/main/java/teammates/common/util/TimeHelper.java

```diff
@@ -502,14 +502,14 @@ public static LocalDateTime parseLocalDateTime(String dateTimeString, String pat
      * Parses a `LocalDateTime` object from separated date, hour and minute strings.
      *
      * <p>required parameter format:
-     * date: dd/MM/yyyy  hour: H   min:m
+     * date: EEE, dd MMM, yyyy  hour: H   min:m
      * Example: If date is 01/04/2014, hour is 23, min is 59.
```

Comment:
You should also update the `Example` line to reflect the new expected format since we cannot accept `01/04/2014` anymore :smile:

Since you're touching the javadoc anyway, you can update this javadoc to meet our [conventions](https://oss-generic.github.io/process/codingStandards/CodingStandard-Java.html#comments) and use the `@param` annotations for the javadoc, e.g.
```
@param date  date in format "EEE, dd MMM, yyy"
@param hour  hour-of-day (0-23)
@param min   minute-of-hour (0-59)
```

You can use the patterns table from the [DateTimeFormatter](https://docs.oracle.com/javase/8/docs/api/java/time/format/DateTimeFormatter.html) docs to help you write this.

You can also update the description of the method since it's specifically for a sessions-related page.